presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -163,16 +162,7 @@ else if (element instanceof LikeClause) {
         Map<String, Object> finalProperties = combineProperties(statement.getProperties().keySet(), properties, inheritedProperties);
 
         ConnectorTableMetadata tableMetadata = new ConnectorTableMetadata(tableName.asSchemaTableName(), ImmutableList.copyOf(columns.values()), finalProperties, statement.getComment());
-
-        try {
-            metadata.createTable(session, tableName.getCatalogName(), tableMetadata);
-        }
-        catch (PrestoException e) {
-            if (!e.getErrorCode().equals(ALREADY_EXISTS.toErrorCode()) || !statement.isNotExists()) {
-                throw e;
-            }
-        }
-
+        metadata.createTable(session, tableName.getCatalogName(), tableMetadata, statement.isNotExists());
```

Comment:
Based on discussion with @electrum, let's revert this. Add a comment that this is needed when connectors don't handle the ignoreExisting flag, which is optional.